rector-config-docroot-dir.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Core\Configuration\Option;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->import(__DIR__ . '/vendor/palantirnet/drupal-rector/config/drupal-8/drupal-8-all-deprecations.yml');
+
+    $parameters = $containerConfigurator->parameters();
+
+    $parameters->set(Option::AUTOLOAD_PATHS, ['docroot/core', 'docroot/core/modules', 'docroot/modules', 'docroot/profiles']);
+    $parameters->set(Option::EXCLUDE_PATHS, ['*/upgrade_status/tests/modules/*']);
```

Comment:
We'll want to add another commented out line here for people to exclude the test folders.